vignettes/markdown.Rmd

```diff
@@ -141,33 +148,52 @@ URLs inside angle brackets are also automatically converted to hyperlinks:
+  * `[text][ref]`: Link text differs from the target.
+
+with variations depending on whether `ref` is in the same package and whether the link is automatically typeset as code. Use backticks inside the square brackets to typeset the link text as code, when it would not be otherwise:
+
+  * ``[`ref`]``
+  * ``[`text`][ref]``
+
++-----------------------+------------------------+--------------------------+
+| Examples              | Help topic\            | Notes                    |
+|                       | for what?              |                          |
++=======================+========================+==========================+
+| `[func()]`            | a function in\         | Always typeset as code.\            |
+|                       | same package           | In Rd: `\code{\link[=func]{func()}` | 
++-----------------------+------------------------+--------------------------+  
+| `[object]`\           | a topic in\            | Use for a topic where you documented\ |
+| ``[`object`]``        | same package           | `NULL` and set its name via `@name`,\ |
+|                       |                        | e.g., a class, dataset, or theme.\    | 
+|                       |                        | Bring your own backticks.\            |
+|                       |                        | In Rd: `\link{object}`                |
++-----------------------+------------------------+--------------------------+
+| `[text][func()]`\     | a function or topic\   | Use when you want to provide your\               |
+| `[text][object]`\     | in same package        | own link text.\                                  |
+| `[abc][abc-class]`    |                        | Bring your own backticks.\                       |
+|                       |                        | In Rd: `\link[=func]{text}`,                     |
+|                       |                        | `\link[=obj]{text}`, or `\link[=abc-class]{abc}` |
++-----------------------+------------------------+--------------------------+
+| `[pkg::func()]`       | a function in `pkg`    | Always typeset as code.\                     |
+|                       |                        | In Rd: `\code{\link[pkg:func]{pkg::func()}}` |
++-----------------------+------------------------+--------------------------+
+| `[pkg::object]`       | a topic in `pkg`       | Use for a non-function object in `pkg`\ |
+|                       |                        | Bring your own backticks.\              |
+|                       |                        | In Rd: `\link[pkg:object]{pkg::object}` |
++-----------------------+------------------------+--------------------------+
+| `[text][pkg::func()]` | a function or topic\   | Use when you want to provide your\ |
+| `[text][pkg::object]` | in `pkg`               | own link text.\                    |
+|                       |                        | Bring your own backticks.\         |
```

Comment:
Again, `text` is almost never code IMO.

Reply:
Yeah I am only pointing out that you must use the backticks if you want `\code{...}`. I guess that is confusing.